packages/css/tooltip.css

```diff
@@ -11,8 +12,33 @@
   padding: var(--dsc-tooltip-padding);
 
   @composes ds-body-text--short-xs from './base/base.css';
-}
 
-.ds-tooltip__arrow {
-  fill: var(--dsc-tooltip-background);
+  &::before {
+    content: '';
+    background: var(--dsc-tooltip-background);
+    box-sizing: border-box;
+    height: var(--dsc-tooltip-arrow-size);
+    width: var(--dsc-tooltip-arrow-size);
+    position: absolute;
+    left: var(--ds-tooltip-arrow-x);
+    top: var(--ds-tooltip-arrow-y);
+    translate: -50% -50%;
+    rotate: 45deg;
+  }
+
+  &[data-placement='top']::before {
+    top: 100%;
+  }
+
+  &[data-placement='left']::before {
+    left: 100%;
+  }
+
+  &[data-placement='right']::before {
+    left: 0;
+  }
+
+  &[data-placement='bottom']::before {
+    top: 0;
+  }
```

Comment:
I copied this from `Popover`, but would it be better to do this in the floating function?